src/services/classes.js

```
@@ -122,8 +123,27 @@ module.exports = class ClassesHelper {
         options
       );
 
+
       if (classes.count > 0) {
-        results.data = classes.rows;
+        const organization_ids = [...new Set([...classes?.rows?.map(classItem => classItem.organization_id)])] || [];
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)